Readme.md

```diff
@@ -141,12 +141,13 @@ see where it's stored) if you don't plan to reinstall Dolphin.
 * -A, --audio_emulation=<str> Low level (LLE) or high level (HLE) audio
 
 Available DSP emulation engines are HLE (High Level Emulation) and
-LLE (Low Level Emulation). HLE is fast but often less accurate while LLE is
+LLE (Low Level Emulation). HLE is fast but can be less accurate while LLE is
```

Comment:
I think "often" is still a better word here.

Reply:
Is HLE incorrect enough to be "often" less accurate than LLE, though?
Maybe something like
```suggestion
LLE (Low Level Emulation). HLE is fast but is less accurate than LLE, which is
```